### DISCUSSION TITLE
Line numbers in code block

### DISCUSSION THREAD

**Discussion by spring-haru** [executablebooks/meta#343] [Q&A]
I tried to find  how to add line numbers in code block, but no chance. Is there a global option to turn it on?

**Comment by rajk-cerebras** [comment]
Have you tried this? This works for me. https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html?highlight=line%20number#showing-code-examples

**Reply by spring-haru** [reply]
I guess you would be right if relevant code blocks were in a `.md` file. I use `.ipynb` files in many cases and want to know how to add line numbers in code blocks in those files. For example, using a tag. I tried `linenos` in tag, but it did not work, as you might have expected. Thanks anyway.

**Reply by choldgraf** [reply]
I opened up an issue to discuss adding this with input cells: https://github.com/executablebooks/MyST-NB/issues/324
